### PR TITLE
Strategy for SDL and voting power

### PR DESCRIPTION
**Pull request by nguyenaiden**
Added strategies for:
-  SDL + locked SDL balance
- veSDL voting power

### CODE REVIEW

File: src/strategies/saddle-vesdl-voting-power/index.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  addresses.forEach((address) =>
+    multi.call(address, options.address, 'getPriorVotes', [address, blockTag])
+  );
+  const result: Record<string, BigNumberish> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address, balance]) => [
+      address,
+      parseFloat(formatUnits(balance, options.decimals))
+    ])
```
Comment (ChaituVR) [inline src/strategies/saddle-vesdl-voting-power/index.ts:32]: Hey @nguyenaiden For this strategy, can you try the `contract-call` strategy and see if it works? Thanks :)

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hey I think you can use `erc20-balance-of` strategy and two `contract-call` strategies, one for locked and one for getPriorVotes

Result will be sum of these three strategies

File: src/strategies/saddle-vesdl-voting-power/index.ts (hunk 1 above)
Comment (nguyenaiden) [reply src/strategies/saddle-vesdl-voting-power/index.ts:32]: @ChaituVR Could you explain this further? I'm not sure what you meant by trying the `contract-call` strategy.

Comment (ChaituVR) [reply src/strategies/saddle-vesdl-voting-power/index.ts:32]: I am this strategy https://snapshot.org/#/strategy/contract-call , you can directly call a method from a contract, try playing with it in the playground here https://snapshot.org/#/playground/contract-call

**Pull request closed without merging**